### PR TITLE
feat(dashboard): Backfill aggregate and column data

### PR DESCRIPTION
**Pull request by shruthilayaj**
Migration to backfill columns and aggregates fields with
exisiting fields data.

### CODE REVIEW

File: src/sentry/migrations/0276_backfill_dashboard_widget_query_columns_aggregates.py [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+        fields = widget_query.fields or []
+        columns = []
+        aggregates = []
+
+        for field in fields:
+            if is_aggregate(field):
+                aggregates.append(field)
+            else:
+                columns.append(field)
+
```
Comment (wmak) [inline src/sentry/migrations/0276_backfill_dashboard_widget_query_columns_aggregates.py:50]: Do we need to set `widget_query.columns = columns` here?

**Review by wmak — APPROVED** [review]
Looks good to me, to confirm the `is_equation` and `is_aggregate` functions are supposed to match the ones we already have on the frontend right?

File: src/sentry/migrations/0276_backfill_dashboard_widget_query_columns_aggregates.py (hunk 1 above)
Comment (wedamija) [reply src/sentry/migrations/0276_backfill_dashboard_widget_query_columns_aggregates.py:50]: Also need to do with `aggregates`. Could we just operate on the values in `widget_query` directly?

Comment (shruthilayaj) [reply src/sentry/migrations/0276_backfill_dashboard_widget_query_columns_aggregates.py:50]: oh wow rip

**Review by wedamija — COMMENTED** [review]
This table is 40k, should be fine to run as not dangerous. 

It's usually worth testing that a migration does what you intend, you can use this code to run a local test pretty quickly (Don't need to commit it)

https://github.com/getsentry/sentry/blob/9ffdd1b6d88a76f7b827fdbd189eb41a556584c4/tests/sentry/migrations/test_0223_semver_backfill_2.py#L13

**Comment by shruthilayaj** [comment]
> This table is 40k, should be fine to run as not dangerous.
> 
> It's usually worth testing that a migration does what you intend, you can use this code to run a local test pretty quickly (Don't need to commit it)
> 
> https://github.com/getsentry/sentry/blob/9ffdd1b6d88a76f7b827fdbd189eb41a556584c4/tests/sentry/migrations/test_0223_semver_backfill_2.py#L13

yup, I logged to see that the fields were getting split correctly into columns and aggregates, I just forgot to to see if they were actually being saved to the dashboard widget query, total brain slip

**Comment by shruthilayaj** [comment]
ran it locally,
```
In [1]: from sentry.models import DashboardWidgetQuery

In [2]: DashboardWidgetQuery.objects.filter(aggregates__isnull=False).count()
Out[2]: 61

In [3]: DashboardWidgetQuery.objects.filter(columns__isnull=False).count()
Out[3]: 61

In [4]: DashboardWidgetQuery.objects.first().columns
Out[4]: []

In [5]: DashboardWidgetQuery.objects.first().aggregates
Out[5]: ['count()']

In [6]: DashboardWidgetQuery.objects.all().count()
Out[6]: 61
```

**Review by wedamija — APPROVED** [review]
Migration looks safe to me

**Pull request merged**